cola/ace_generator.py

```diff
@@ -0,0 +1,90 @@
+from .ce_module.ce_models import DiCE
+from .model import Model
+from .data import Data
+from .policy_module import policy
+import numpy as np
+import pandas as pd
+
+class COLA:
+    def __init__(self, ml_model=None, data=None, counterfactual_algorithm=None, policy_name="CF_OTMatch", Avalues_method="max", limited_actions=None):
```

Comment:
I noticed that you have declared type for other class. I think it is good to do it for all your functions. E.g.

ml_model : Model